test/setup.ts

```diff
@@ -1,11 +1,18 @@
-// @ts-expect-error
-window.ResizeObserver ??= function ResizeObserver(callback: () => void) {
-  callback();
-
-  return {
-    observe() {},
-    disconnect() {}
-  };
+import 'core-js/stable';
```

Comment:
TS complains without an import/export, so I added this. It doesn't hurt to have I guess.

```
test/setup.ts:1:1 - error TS1208: 'setup.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.

1 window.ResizeObserver ??= class {
  ~~~~~~
```